js/samples/04.ai.d.chainedActions.listBot/package.json

```diff
@@ -19,19 +19,20 @@
         "url": "https://github.com"
     },
     "dependencies": {
+        "@microsoft/teams-ai": "~0.1.0-preview.2",
         "botbuilder": "^4.19.3",
-        "@microsoft/teams-ai": "~0.1.0-preview.1",
         "dotenv": "^8.2.0",
         "replace": "~1.2.0",
-        "restify": "~8.6.0"
+        "restify": "~10.0.0"
```

Comment:
Did just the bump work? I have a PR open for bumping restify, but it also requires bot code changes.

Reply:
https://github.com/microsoft/teams-ai/pull/198

Reply:
no the `tsconfig` needs to also target `>= es2017` so it will use `async` functions, restify freaks out over it with new versions.

Reply:
but we can close this in favor of https://github.com/microsoft/teams-ai/pull/226